go/teams/service_helper.go

```diff
@@ -1546,11 +1547,34 @@ func ListRequests(ctx context.Context, g *libkb.GlobalContext, teamName *string)
 		return nil, err
 	}
 
-	joinRequests := make([]keybase1.TeamJoinRequest, len(arList.Requests))
+	var (
+		joinRequests  = make([]keybase1.TeamJoinRequest, len(arList.Requests))
+		requesterUIDs = make([]keybase1.UID, len(arList.Requests))
+	)
 	for i, ar := range arList.Requests {
+		username := libkb.NewNormalizedUsername(ar.Username)
+		uid := libkb.GetUIDByNormalizedUsername(g, username)
+
+		requesterUIDs[i] = uid
 		joinRequests[i] = keybase1.TeamJoinRequest{
 			Name:     ar.FQName,
-			Username: libkb.NewNormalizedUsername(ar.Username).String(),
+			Username: username.String(),
+			Ctime:    keybase1.ToUnixTime(ar.Ctime),
+		}
+	}
+
+	packages, err := g.UIDMapper.MapUIDsToUsernamePackages(ctx, g, requesterUIDs,
+		defaultFullnameFreshness, 10*time.Second, true)
+	if err != nil {
```

Comment:
From MapUIDsToUsernamePackages
```
    // *NOTE* that this function can return useful data and an error. In this
	// regard, the error is more like a warning. But if, for instance, the
	// mapper runs out of time budget, it will return the data
```
maybe we do not need to error out on this.

Reply:
correct, it can error out on API call but still return cached data